doc/escapable_handle_scope.md

```diff
@@ -40,16 +40,16 @@ Napi::EscapableHandleScope Napi::EscapableHandleScope::New(napi_env env, napi_ha
 
 Returns a new `Napi::EscapableHandleScope` instance which wraps the
 napi_escapable_handle_scope handle passed in. This can be used
```

Comment:
Could you use backtick for napi_escapable_handle_scope? From napi_escapable_handle_scope to `napi_escapable_handle_scope`